frontends/web/src/routes/market/components/buysell.tsx

```diff
@@ -29,31 +29,31 @@ import { A } from '@/components/anchor/anchor';
 import { InfoButton } from '@/components/infobutton/infobutton';
 import { getConfig } from '@/utils/config';
 import { ActionableItem } from '@/components/actionable-item/actionable-item';
-import { ExchangeProviders } from '@/routes/exchange/components/exchange-providers';
-import style from '../exchange.module.css';
+import { VendorDeals } from '@/routes/market/components/vendor-deals';
+import style from '../market.module.css';
 
 type TProps = {
   accountCode: string;
   selectedRegion: string;
-  goToExchange: (exchange: string) => void;
+  goToVendor: (vendor: string) => void;
   showBackButton: boolean;
-  action: exchangesAPI.TExchangeAction;
+  action: marketAPI.TMarketAction;
   setInfo: (info: TInfoContentProps) => void;
 }
 
 export const BuySell = ({
```

Comment:
Nit: feels like it is time to rename this component, maybe just Deals or smth similar?